src/openvslam/util/stereo_rectifier.cc

```diff
@@ -12,21 +12,22 @@ stereo_rectifier::stereo_rectifier(const std::shared_ptr<openvslam::config>& cfg
     : stereo_rectifier(cfg->camera_, cfg->yaml_node_) {}
```

Comment:
I think the following is the minimum change.

```suggestion
    : stereo_rectifier(cfg->camera_, cfg->yaml_node_["StereoRectifier"]) {}
```

Reply:
In addition, it is preferable to use `yaml_optional_ref` to ensure that default values are used.

```suggestion
    : stereo_rectifier(cfg->camera_, openvslam::util::yaml_optional_ref(cfg->yaml_node_, "StereoRectifier")) {}
```